src/logger.ts

```diff
@@ -0,0 +1,29 @@
+import { IContext, IJSLoggerDefaultsType, ILogger, ILoggerOpts, ILogLevel, JSLoggerExportType, LogHandler } from "./datatypes";
+export * from "./datatypes";
+import { globalLogger, JSLoggerDefaults } from "./JSLoggerDefaults";
+import { bind, defineLogLevel } from "./utils";
+
+const boundGlobalLoggerFunctions: ILogger = {
+  enabledFor: bind(globalLogger, globalLogger.enabledFor),
+  trace: bind(globalLogger, globalLogger.trace),
+  debug: bind(globalLogger, globalLogger.debug),
+  time: bind(globalLogger, globalLogger.time),
+  timeEnd: bind(globalLogger, globalLogger.timeEnd),
+  info: bind(globalLogger, globalLogger.info),
+  warn: bind(globalLogger, globalLogger.warn),
+  error: bind(globalLogger, globalLogger.error),
+  log: bind(globalLogger, globalLogger.info),
+  getLevel: bind(globalLogger, globalLogger.getLevel),
+  setLevel: () => { /* */ }, // will be overwritten
+};
+
+export const Logger: JSLoggerExportType = { ...boundGlobalLoggerFunctions, ...JSLoggerDefaults };
+
+// For those that are at home that are keeping score.
+// Logger.VERSION = "1.4.1";

```

Comment:
We should continue to export the version here (or remove this an make a breaking change in 2.0)

Reply:
Yeah, I think we can definitely keep the version. Just need to add it to the type.